src/Tools/dotnet-monitor/DiagnosticPortOptions.cs

```diff
@@ -10,13 +14,20 @@ namespace Microsoft.Diagnostics.Tools.Monitor
 {
     public class DiagnosticPortOptions
     {
+        [Description(@"In Connect mode, dotnet-monitor connects to the application for diagnostics. In Listen
+mode the application connects to dotnet-monitor via " + nameof(EndpointName))]
+        [DefaultValue(DiagnosticPortOptionsDefaults.ConnectionMode)]
         public DiagnosticPortConnectionMode? ConnectionMode { get; set; }
 
+        [Description(@"In Listen mode, specifies the name of the named pipe or unix domain socket to use for connecting
```

Comment:
nit: "Listen" should be surrounded by single quotes. And end the sentence with a period.